tests/src/test/scala/system/basic/WskRuleTests.scala

```diff
@@ -27,6 +27,8 @@ import common.WskTestHelpers
 import spray.json._
 import spray.json.DefaultJsonProtocol._
 import java.time.Instant
+import scala.concurrent.duration.DurationInt
+import scala.language.postfixOps
```

Comment:
Please drop `postfixOps` and use standard notation of `60.seconds`. You'll get super weird compile errors when using `postfixOps`.

To import duration, just do `import scala.concurrent.duration._`